src/databaseQueries/deletionAuditLogs.js

```
@@ -0,0 +1,35 @@
+		return new Promise(async (resolve, reject) => {
+			try {
+				let deletedData = await database.models.deletionAuditLogs.insertMany(logs)
+				return resolve(deletedData)
+			} catch (error) {
+				return reject({
+					status: error.status || HTTP_STATUS_CODE.bad_request.status,
+					message: error.message || HTTP_STATUS_CODE.bad_request.message,
+					errorObject: error,
+				})
+			}
+		})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)